examples/meals.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type Meal struct {
+	Title   string
+	Comment string
+}
+
+func main() {
+	var meals = []Meal{
+		{Title: "Bread", Comment: "Good, not so healthy"},
+		{Title: "Eggs", Comment: "Healthy"},
+		{Title: "Apple", Comment: ""},
+		{Title: "Burger", Comment: "Really?"},
+	}
+
+	answers := struct {
+		Meal string
+	}{}
+
+	titles := make([]string, len(meals))
+	for i, m := range meals {
+		titles[i] = m.Title
+	}
+	var qs = []*survey.Question{
+		{
+			Name: "meal",
+			Prompt: &survey.Select{
+				Message: "Choose a meal:",
+				Options: titles,
+				Description: func(value string, index int) string {
+					return meals[index].Comment
+				},
+			},
+			Validate: survey.Required,
```

Comment:
This isn't a demo for validation features, and since validation isn't necessary here, we could just omit this.